tern/analyze/default/container/run.py

```diff
@@ -81,14 +81,9 @@ def execute_image(args):
             # Add an image origin here
             full_image.origins.add_notice_origin(
                 formats.docker_image.format(imagetag=image_string))
-            # Set up for analysis
-            setup(full_image)
-            # analyze image
-            cimage.analyze(full_image, args.redo, args.driver, args.extend)
-            # report out
-            report.report_out(args, full_image)
-            # clean up
-            teardown(full_image)
+            # Doing analysis
+            dcom.default_common_procedure(
+                full_image, args.redo, args.driver, args.extend, args=args, image_file=True)
```

Comment:
I think this is going to trigger a prospector error with too many arguments.

Reply:
Oh I see, because you call it again in `run.py` for a Docker. Hmm... prospector is still going to complain about number of args I think and I don't want to merge technical debt in here.

Reply:
I think I can get rid of image_file here, only checking whether args are None or not will help to recognize that function is called from `dockerfile/run.py` or from `container/run.py`, hence image_file argument is not required. I think this will remove the technical debt that you are taking @rnjudge.